app/components/SearchBox.tsx

```diff
@@ -64,8 +72,8 @@ const SearchBox: React.FunctionComponent<Props> = ({
         ObjectFieldTemplate={FormObjectFieldTemplate}
         onSubmit={handleSubmit}
       >
-        <div className="search-actions">
-          <ButtonGroup style={{marginLeft: '20px'}}>
+        <div className="search-actions" style={{textAlign: 'right'}}>
+          <ButtonGroup>
```

Comment:
@pbastia You could add some `marginTop` to `.search-actions` here to bring the vertical alignment into line with the other inputs. 

re: my offline comment in standup, it's best practise to stick with CSS only to adjust appearance instead of adding new, empty HTML DOM elements just for the purpose of spacing (and which won't be used for content).